packages/core/package.json

```diff
@@ -18,8 +18,19 @@
   "engines": {
     "node": "^14.0.0 || ^16.0.0 || ^18.0.0"
   },
-  "main": "dist/src/index.js",
-  "types": "dist/src/index.d.ts",
+  "main": "./dist/src/index.js",
+  "types": "./dist/src/index.d.ts",
+  "exports": {
+    ".": "./dist/src/index.js",
+    "./helpers": "./dist/src/helpers.js"
+  },
+  "typesVersions": {
+    "*": {
+      "helpers": [
+        "./dist/src/helpers"
+      ]
+    }
+  },
```

Comment:
What does this do?

Reply:
It is what is allowing the reading of type information for the `@ignored/ignition-core/helpers` entry point.

It is why `@hardhat-ignition` is able to resolve the type information. I haven't checked its impact on `.js.map` and `.d.ts.map` files though (vscode was navigating but let me double check)